pkg/secrets/k8s_secrets_storage/mocks/k8s_secrets_client.go

```diff
@@ -2,12 +2,13 @@ package mocks
 
 import (
 	"errors"
+	v1 "k8s.io/api/core/v1"
```

Comment:
@saprette can you add please a blank line before this one?

Reply:
done